tools/system_libs.py

```diff
@@ -775,6 +775,9 @@ def get_files(self):
           'thrd_sleep.c',
           'thrd_yield.c',
           'call_once.c',
+          'tss_create.c',
+          'tss_delete.c',
+          'tss_set.c',
```

Comment:
Perhaps these C11 thread functions should only be included if `self.is_mt` is true (see line 758)? It seems that these are now always included, even when not targeting pthreads.

Reply:
All these functions are built on top the lower level pthread functions which are already stubbed out  in `./system/lib/pthread/library_pthread_stub.c`.. so I think its fine to include these as is.. they will fail at runtime, just like the pthread function on which they are built.